config/config_defaults.go

```diff
@@ -732,6 +732,8 @@ const CGRATES_CFG_JSON = `
 	"max_wait_connection": "2s",			// maximum duration to wait for a connection to be retrieved from the pool
 	"active_session_delimiter": ",",		// delimiter for 'show channels' responses and requests
 	"route_profile": false,					// attaches RouteProfileID to RouteIDs which are sent as reply to freeswitch agent authorization requests
+	"transfer_extension": "9999",				// prepaid calls will be transferred to this extension in authorization 
```

Comment:
Please rename to sched_transfer_extension and remove sched_hangup variable. If sched_transfer_extension will be empty, should be equivalent to "sched_hangup": true.